tests/tests_pytorch/graveyard/test_profiler.py

```diff
@@ -0,0 +1,28 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+
+def test_v2_0_0_base_profilers():
```

Comment:
Why here is it 2.0 when the warning says 1.9?

Reply:
In my understanding, as #15049 suggests, we keep raising exceptions with helpful messages until 2.0 regardless of the version that API removal took place in.